Triphouse-project/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="./style.css"/>
+    <title>What do we offer</title>
+</head>
+<body>
+    <div class="wrapper">
+        <h2>What do we offer</h2>
+        <div class="wrapper-line">
+        </div>
+        <div class="wrapper-offer">
+            <div class="wrapper-item">
+                <img src="../img/Vector.jpg" alt="headphones" width="75px" height="91.67px">
+                <p>Support 24/7</p>
+            </div>
+            <div class="wrapper-item">
+                <img src="../img/Vector (1).jpg" alt="message" width="83.33px" height="83.33px">
```

Comment:
See the previous comment.

Reply:
registered the height and width of the pictures in the css file, set the same dimensions for all pictures